src/main/scheduler/scheduler.go

```diff
@@ -57,8 +57,8 @@ func (s *Scheduler) liveLogTag() string {
 	return "TEST"
 }
 
-func (s *Scheduler) notifyOrder(order model.Order, transactionIds []string) error {
-	if s.notifier == nil || s.config.NotifyEmailAddress == "" {
+func (s *Scheduler) notifyOrder(order model.Order, transactionIds []string) []error {
+	if s.notifiers == nil {
```

Comment:
Does this work as expected? I would have thought you'd have to check `len(s.notifiers)`